social_core/backends/oauth.py

```diff
@@ -494,7 +494,7 @@ def get_code_verifier(self):
 
     def generate_code_challenge(self, code_verifier, challenge_method):
         method = challenge_method.lower()
-        if method == "s256":
+        if method in ["s256", "S256"]:
```

Comment:
„method“ should be lowercase here due to the previous line 496. Why would this change be needed?

Reply:
You're right. Removed this change.